.github/workflows/deploy_agent.yml

```diff
@@ -10,33 +10,40 @@ on:
         description: 'Run ID of the Release Workflow (all_solutions.yml) that was triggered by creating a Release in GitHub.  ID can be found in URL for run.'
         required: true
       deploy:
-        description: 'If "true", deploy the artifacts. If "false", do everything except deploy.'
+        type: boolean
+        description: Deploy Build Artifacts. (If not selected, do everything except deploy.)
         required: true
-        default: 'false'
+        default: true
```

Comment:
This used to default to `false`, is there a reason we are changing it to `true`?

Reply:
Because we always end up setting it to `true` when we run the workflow during a release. Might as well default everything to the value it's normally going to be anyway.

Reply:
We never run Deploy automatically, right? That would be my only concern with changing the defaults.

Reply:
No, it is only ever invoked manually. Note that the only trigger in the workflow is `workflow_dispatch`

Reply:
I'm fine with changing the defaults if we think it is safe, but I would like the change called out in the PR description so that we know it was done intentionally.